src/content-scanner/readScannerUrlConfig.ts

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
This file start by lower case but other file start by upper case

Reply:
Yes. This file only exports a function with that name `readScannerUrlConfig` while the other files export components or classes like for instance `ContentScanner`.

This file is going to be removed anyway.